lib/my_inject.rb

```diff
@@ -0,0 +1,24 @@
+class Array
+  def my_inject accum=first, sym="", &bloc
```

Comment:
Assignment Branch Condition size for my_inject is too high. [15.3/15]<br>Surrounding space missing in default value assignment.